### ISSUE TITLE
Integers serialized as string when converting elkg -> elkt

### ISSUE THREAD

**Issue by uruuru** [eclipse/elk#231]
Converting the following elkg file to elkt results in the integer value being serialized as a string: 

```
<?xml version="1.0" encoding="ASCII"?>
<graph:ElkNode xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:graph="http://www.eclipse.org/elk/ElkGraph">
  <properties key="org.eclipse.elk.layered.thoroughness" value="2"/>
  <children identifier="n1"/>
</graph:ElkNode>
```

```
graph G1
thoroughness: "2"
node n1
```

**Comment by spoenemann** [comment]
It's not really a bug because the string variant works as well, but I'll see what we can do to serialize the most appropriate form when possible.

**Comment by le-cds** [comment]
@spoenemann If you manage to fix this for the 0.3.0 release, that'd be great. Otherwise, simply remove the milestone from the ticket. ;)
